source/MRMesh/MRICP.h

```diff
@@ -67,13 +67,15 @@ struct IPointPairs
 
     virtual const ICPPairData& operator[]( size_t ) const = 0;
     virtual ICPPairData& operator[]( size_t ) = 0;
+    virtual size_t size() const = 0;
```

Comment:
why is it needed?

Reply:
It is needed for allocating IPointPairs array with known size for c and c# bindings